src/main/java/org/openrewrite/quarkus/QuarkusExecutionContextView.java

```diff
@@ -61,4 +68,25 @@ public List<String> getDefaultApplicationConfigurationPaths() {
                 Arrays.asList("**/application.{properties,yaml,yml}", "**/META-INF/microprofile-config.properties")
         );
     }
+
+    public boolean isQuarkusConfigFile(Tree tree, @Nullable List<String> pathExpressions) {
+        if (!(tree instanceof Properties.File || tree instanceof Yaml.Documents)) {
+            return false;
+        }
+        List<String> expressions = pathExpressions != null ? pathExpressions : Collections.emptyList();
+        if (expressions.isEmpty()) {
+            // If not defined, get reasonable defaults from the execution context.
+            expressions = getDefaultApplicationConfigurationPaths();
+        }
+        if (expressions.isEmpty()) {
+            return true;
+        }
```

Comment:
I don't quite understand this second check if expressions is empty, seeing how it's set on line 79. Could you give me some context so I understand better?

Reply:
This is really not necessary, since the previous call will always return a valid list. I'll fix it.